cleanslate/doma/views.py

```diff
@@ -187,8 +201,8 @@ def create_chore(request):
                 created_on = timezone.now(),
                 deadline = form.cleaned_data['deadline']
             )
-            chore.save()
-            messages.success(request, 'You successfully created a chore.')
+            if chore.save():
```

Comment:
Why did you make this change?

Reply:
That's because I was having the same problem with weird button behaviour here too, even when you were hitting 'cancel' on the form it was popping up with a success message when it shouldn't have been, this fixes that.. sort of